src/app/components/Search/search-utils.ts

```diff
@@ -49,6 +50,13 @@ export const searchSuggestionTerms = {
       suggestedTokenFragment: 'USD',
     },
     cipher: undefined,
+    pontusx: {
```

Comment:
We don't wan't to show this runtime in search results for now.

Reply:
Good point, will fix that.

Reply:
> We don't wan't to show this runtime in search results for now.

Fixed.

Reply:
I would make this `undefined` if possible, the suggested block/tx will show up as not found anyway.

Reply:
We could make this `undefind` (after extending the type a little bit), but the problem is that if don't have search suggestions for a layer, we will just use mainnet sapphire data. So it doesn't help. I think we can leave it as it is now, and update with meaningful suggestions when we have actually started indexing the data.